troubleshooting/diagnostics.md

```diff
@@ -47,13 +47,30 @@ In this example, the OAK-D node is not running, so the camera topics are not bei
 
 For systems experiencing high CPU usage issues or limited bandwidth, it can help to disable the diagnostics whenever they are not actively in use.
 
-{% tabs navigation %}
-{% tab navigation galactic %}
+{% tabs disable_diagnostics %}
+{% tab disable_diagnostics galactic %}
+```warning
+**ROS 2 Galactic is no longer supported.** Please consider upgrading to a newer release
+```
 
 Instructions are not currently available for Galactic.
 
 {% endtab %}
-{% tab navigation humble %}
+{% tab disable_diagnostics humble %}
+
+
+To disable diagnostics, SSH into your TurtleBot 4 and run the turtlebot4 setup tool:
+
+```bash
+turtlebot4-setup
+```
+
+Navigate to 'Bash Setup' in the 'ROS Setup' menu, then change the `DIAGNOSTICS` setting to `False`.
+Save the settings, then apply settings in the main menu.
```

Comment:
Superfluous new line or maybe this should be two paragraphs (flagged because the space was removed as part of that other discussion)